src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -1 +1,5 @@
 ; Please do not edit this file manually, it should only be updated through code fix application.
+### New Rules
```

Comment:
@mavasani FYI, the code that generates this need to be updated to respect markdown rules, i.e. leave blank lines around headings.

See https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines

Reply:
So my fix is simply to insert a blank line manually? Thanks for the heads up :)

Reply:
I believe this shouldn't be edited manually.

Reply:
Yes, but otherwise I do not see a fix I could apply. I think updating the template is a little out of scope for this pr is it not?

Reply:
@Mrnikbobjeff Yes it's indeed out of scope for this PR.
I was just noting this to @mavasani to consider if he wants to fix that.

Reply:
Yes, please don't update this file manually and file a separate issue to fix the code fix that updates the file.